### ISSUE TITLE
GT5U creative inventory crashes game when flasks should be rendered

### ISSUE THREAD

**Issue by Ubus99** [Blood-Asp/GT5-Unofficial#1564]
The Mod crashes whenever GT_FlaskRenderer.java:82 is called.
[crash-2020-05-26_09.55.57-client.txt](https://github.com/Blood-Asp/GT5-Unofficial/files/4680669/crash-2020-05-26_09.55.57-client.txt)

**Comment by Ubus99** [comment]
Bug only happens with IC2 Classic

**Comment by Ubus99** [comment]
IC2 classic has no ic2.core.util.DrawUtil class...

**Comment by draknyte1** [comment]
I'll take a look at it, see how much of that code can be rewritten into GT itself.
Alternatively You could ask Speiger to implement an equivalent into IC2C.

**Comment by Ubus99** [comment]
Makes sense, I just assumed since IC2Classic was the default in the build.gradle that it was supported

**Comment by MauveCloud** [comment]
@draknyte1 It looks to me like GT5u already has a class that could be used instead: https://github.com/Blood-Asp/GT5-Unofficial/blob/unstable/src/main/java/gregtech/common/render/GT_RenderUtil.java
